Hexbin map and percentages bar chart

TobiasBak opened the pull request:
Data cleaner now also adds a percentage of accidents in state column

TobiasBak committed:
Initial hexbin map, values can be tweaked etc.

TobiasBak committed:
Radio buttons for adjusting hexagon sizes

TobiasBak committed:
Added radio buttons and removed ocean dump

TobiasBak committed:
Added some guidance when using the map

thor2304 committed:
Alternate colorscale

TobiasBak committed:
Fixed so percentages works

thor2304 committed:
Improved the showcase of colorbins

KevinTorp commented on src/Core/data_cleaning/data_cleaner.py:
```diff
@@ -131,18 +131,19 @@ def main():
 
-    # Drop rows where longitude is above -45
+    # Drop rows where longitude is higher than -45
     df = df[df['Longitude'] < -45]
```
Didnt you fix this somehow else - it still removes 10.000 rows +?

thor2304 committed:
Includes nan latitude longitude when filtering

thor2304 committed:
Merged new colorscale into the map

thor2304 committed:
Using the correct sum for the aggregater

thor2304 committed:
Merge branch 'main' into dev/hexbin_map

thor2304 committed:
Add annotation to the year change plot

thor2304 committed:
years are only whole numbers

KevinTorp committed:
Merge branch 'main' into dev/hexbin_map

KevinTorp committed:
Removed print statements spamming console

KevinTorp committed:
Quick fix formatting decimals for color  bar on for map chart

KevinTorp committed:
Changed decimal formatting to pure int

As there can never be .xxx events I think its better to just remove the decimals, so changed the float to an int
